src/protocol.py

```diff
@@ -168,13 +168,14 @@ def checkSocksIP(host):
 
 def isProofOfWorkSufficient(data,
                             nonceTrialsPerByte=0,
-                            payloadLengthExtraBytes=0):
+                            payloadLengthExtraBytes=0,
+                            recvTime=0):
```

Comment:
Are you planning to write some code which will call this `isProofOfWorkSufficient()` with the `recvTime` kwarg? Now it's hard to imagine how this changes supposed to work.

Reply:
Certainly more code changes are needed in order to take advantage of this new feature. Some of these changes are already written and ready to go.
I am just trying to avoid a large diff that would lead to a long review time on my PR, with the ensuing complications. From past experience, small increments seem to work better for everyone.

Reply:
this could be helpful for airgapped machines, or for running the network thread and objectProcessorThread in different processes/machines in general.

Reply:
This is the original purpose indeed, and then I noticed that object time travel may be possible even in the current version, in rare circumstances. For example, when an objectProcessorQueue backlog is persisted to sqlite on client shutdown: 
https://github.com/Bitmessage/PyBitmessage/blob/74ddda8775156d6b4271284b6902058513c5d3d5/src/class_objectProcessor.py#L102
If the client is restarted after a clock change (or a few days break), the persisted objects will return a false status on this check: 
https://github.com/Bitmessage/PyBitmessage/blob/74ddda8775156d6b4271284b6902058513c5d3d5/src/class_objectProcessor.py#L618
Again, unlikely circumstances but not impossible.